SparseMat unit test failure

On Ubuntu 12.10 (64-bit) using biom-format 1.1.2, numpy 1.5.1, gcc 4.7.2, and Cython 0.18, I get the following unit test failure for SparseMat:

Command:
`python python-code/tests/all_tests.py`

Output:

```
======================================================================
FAIL: test_to_sparsemat (test_sparsemat.SupportTests)
Convert to expected SparseMat types
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jrideout/.virtualenvs/qiime/deps/biom-format-1.1.2/python-code/tests/test_sparsemat.py", line 309, in test_to_sparsemat
    self.assertEqual(obs.items(), exp.items())
  File "/home/jrideout/.virtualenvs/qiime/local/lib/python2.7/site-packages/biom/unit_test.py", line 211, in failUnlessEqual
    (msg or 'Got %s, but expected %s' % (`observed`, `expected`))
AssertionError: Got [((1, 0), 5.0), ((8, 10), -1.23)], but expected [((8, 10), -1.23), ((1, 0), 5.0)]
```

print_biom_python_config.py:

```
System information
==================
          Platform: linux2
Python/GCC version: 2.7.3 (default, Sep 26 2012, 21:51:14)  [GCC 4.7.2]
 Python executable: /home/jrideout/.virtualenvs/qiime/bin/python

Dependency versions
===================
              NumPy version:    1.5.1
biom-format library version:    1.1.2
 biom-format script version:    1.1.2

biom-format package information
===============================
SparseObj type: biom.csmat.CSMat

```

This problem also exists in the latest development version of biom-format.

--- a/python-code/tests/test_sparsemat.py
+++ b/python-code/tests/test_sparsemat.py
@@ -306,7 +306,7 @@ class SupportTests(TestCase):
         exp[(1,0)] = 5
         exp[(8,10)] = -1.23
         obs = to_sparsemat(input, transpose=True)
-        self.assertEqual(obs.items(), exp.items())
+        self.assertEqual(sorted(obs.items()), sorted(exp.items()))
 
         # passing a list of dicts, transpose
         exp = SparseMat(3,2)

